docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>Can we remove <code>.strftime('%Y-%m-%d')</code> and change <code>index=time_range</code> to <code>index=pd.to_datetime(time_range)</code>?</p><p> </p><p>It looks more professional, plot renders x-axis more space efficient and index in pandas dataframe is a proper datetime index.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>great, thanks, it is fixed.</p>